packages/testsuite/cypress/e2e/elytron-oidc-client/test-oidc-security.cy.ts

```diff
@@ -0,0 +1,90 @@
+describe("TESTS: Configuration => Subsystems => Elytron OIDC client", () => {
+  let wildfly: string;
+  let ssoServer: string;
+
+  before(() => {
+    cy.startWildflyContainerSecured("standalone.xml")
+      .then((result) => {
+        wildfly = result as string;
+        cy.log("wildfly server endpoint: " + wildfly);
+      })
+      .then(() => {
+        cy.startSsoContainer("sso_server").then((result) => {
+          ssoServer = result as string;
+          cy.log("sso server endpoint: " + ssoServer);
+
+          cy.executeInWildflyContainer(
+            `"/subsystem=elytron-oidc-client/provider=keycloak:add(provider-url=${ssoServer}/realms/wildfly-infra)"`
+          );
+
+          cy.executeInWildflyContainer(
+            `"/subsystem=elytron-oidc-client/secure-deployment=wildfly-management:add(provider=keycloak,client-id=wildfly-management,principal-attribute=preferred_username,bearer-only=true,ssl-required=EXTERNAL)"`
+          );
+
+          cy.executeInWildflyContainer(
+            `"/subsystem=elytron-oidc-client/secure-server=wildfly-console:add(provider=keycloak,client-id=wildfly-console,public-client=true)"`
+          );
+
+          cy.executeInWildflyContainer(`"reload"`);
+        });
+      });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  it("Test OICD authentization without RBAC", () => {
```

Comment:
The names of the tests are a bit different than in Java. It should be described what is being performed.

In this case something like "Logs in successfully and logs out".